packages/core/src/app/payment/paymentMethod/PaymentMethodV2.tsx

```diff
@@ -70,7 +70,17 @@ const PaymentMethodContainer: ComponentType<
         type: method.type,
     });
 
-    if (!ResolvedPaymentMethod || method.gateway === PaymentMethodId.Mollie) {
+    const shouldUseV1 = (method: PaymentMethod, checkoutState: CheckoutSelectors) => {
```

Comment:
Can we move this method outside the component body? So it's not recreated on every render.